test/test_autograd.py

```diff
@@ -3609,145 +3609,149 @@ def maybe_check_raise(fn, should_raise):
                  should_raise_tuple=(None, None, None))
 
     def test_autograd_simple_views_python(self):
```

Comment:
nit: to make the diff look better here, you can simply rename this to `_do_test_autograd_simple_views_python()` and then create two new tests that uses it.

Reply:
My comment here was to re-use the original `test_autograd_simple_views_python` and rename it `_do_test_autograd_simple_views_python`. So that you don't have to re-indent the whole function. And so github won't see the whole function as changed.

Reply:
oh okay updated!